run-tests.sh

```diff
@@ -89,13 +89,16 @@ lockUser=""
 
 testArgs="--debug --verbose --reporter mocha-junit-reporter"
 #
-# DCF tests are not yet stable enough to run in fence PR's -
+# DCF Google tests are not yet stable enough to run in fence PR's -
```

Comment:
you said it is not stable to run in fence? But I see it will run when `$service == "fence"` line 99 and 103